hs_core/forms.py

```diff
@@ -1258,54 +1264,62 @@ class CoverageSpatialForm(forms.Form):
         ('box', 'Box'),
         ('point', 'Point')
     )
-    #type = forms.CharField(max_length=20, widget=Select(choices=TYPE_CHOICES, attrs={'class': 'select'}))
-    type = forms.ChoiceField(choices=TYPE_CHOICES, widget=forms.RadioSelect(renderer=HorizontalRadioRenderer), label='')
+
+    type = forms.ChoiceField(choices=TYPE_CHOICES,
+                             widget=forms.RadioSelect(renderer=HorizontalRadioRenderer), label='')
     name = forms.CharField(max_length=200, required=False, label='Place/Area Name')
-    projection = forms.CharField(max_length=100, required=False, label='Coordinate System/Geographic Projection')
+    projection = forms.CharField(max_length=100, required=False,
+                                 label='Coordinate System/Geographic Projection')
 
-    east = forms.DecimalField(label='Longitude (WGS 84 decimal degrees)', widget=forms.TextInput())
-    north = forms.DecimalField(label='Latitude (WGS 84 decimal degrees)', widget=forms.TextInput())
+    east = forms.DecimalField(label='Longitude', widget=forms.TextInput())
+    north = forms.DecimalField(label='Latitude', widget=forms.TextInput())
     units = forms.CharField(max_length=50, label='Coordinate Units')
     #elevation = forms.DecimalField(required=False, widget=forms.TextInput())
     #zunits = forms.CharField(max_length=50, required=False, label='Elevation Units', help_text='e.g., meters')
-    northlimit = forms.DecimalField(label='North Latitude (WGS 84 decimal degrees)', widget=forms.TextInput())
-    eastlimit = forms.DecimalField(label='East Longitude (WGS 84 decimal degrees)', widget=forms.TextInput())
-    southlimit = forms.DecimalField(label='South Latitude (WGS 84 decimal degrees)', widget=forms.TextInput())
-    westlimit = forms.DecimalField(label='West Longitude (WGS 84 decimal degrees)', widget=forms.TextInput())
+    northlimit = forms.DecimalField(label='North Latitude', widget=forms.TextInput())
```

Comment:
Wondering whether we should keep ```(WGS 84 decimal degrees)``` to better inform users the default geodetic datum we use?